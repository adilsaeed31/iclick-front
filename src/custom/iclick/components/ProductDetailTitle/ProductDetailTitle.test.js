import React from "react";
import renderer from "react-test-renderer";
import ProductDetailTitle from "./ProductDetailTitle";

test("basic snapshot", () => {
  const component = renderer.create(<ProductDetailTitle title="Title" pageTitle="Subtitle" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("snapshot with only title", () => {
  const component = renderer.create(<ProductDetailTitle title="Title" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("snapshot with only pageTitle", () => {
  const component = renderer.create(<ProductDetailTitle pageTitle="Subtitle" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("snapshot with neither title nor pageTitle", () => {
  const component = renderer.create(<ProductDetailTitle />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
