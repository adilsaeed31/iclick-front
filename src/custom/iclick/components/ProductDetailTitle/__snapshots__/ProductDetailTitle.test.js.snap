// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic snapshot 1`] = `
<div
  className="MuiGrid-item-2 MuiGrid-grid-sm-12-55"
>
  <h1
    className="MuiTypography-root-98 MuiTypography-display2-101 MuiTypography-gutterBottom-116"
  >
    Title
  </h1>
  <h2
    className="MuiTypography-root-98 MuiTypography-title-104 MuiTypography-colorPrimary-119"
  >
    Subtitle
  </h2>
</div>
`;

exports[`snapshot with neither title nor pageTitle 1`] = `null`;

exports[`snapshot with only pageTitle 1`] = `
<div
  className="MuiGrid-item-2 MuiGrid-grid-sm-12-55"
>
  <h2
    className="MuiTypography-root-98 MuiTypography-title-104 MuiTypography-colorPrimary-119"
  >
    Subtitle
  </h2>
</div>
`;

exports[`snapshot with only title 1`] = `
<div
  className="MuiGrid-item-2 MuiGrid-grid-sm-12-55"
>
  <h1
    className="MuiTypography-root-98 MuiTypography-display2-101 MuiTypography-gutterBottom-116"
  >
    Title
  </h1>
</div>
`;
